src/tests/unit/tests/common/stores/base-client-stores-hub.test.ts

```diff
@@ -189,16 +189,7 @@ describe('BaseClientStoresHubTest', () => {
     }
 
     function buildClientStoresHub(stores: BaseStore<TestStoreData>[]): BaseClientStoresHub<TestStoreData> {
-        const builder: (stores) => BaseClientStoresHub<TestStoreData> = theStores => {
-            function ClientStoresHubFake(): void {
-                // tslint:disable-next-line: no-invalid-this
-                BaseClientStoresHub.call(this, theStores);
-            }
-            ClientStoresHubFake.prototype = BaseClientStoresHub.prototype;
-            return new ClientStoresHubFake();
-        };
-
-        return builder(stores);
+        return new BaseClientStoresHub<TestStoreData>(stores);
```

Comment:
This started failing with a TypeError for what was previously only the disabled tslint failure. Wahaj and I looked this over and weren't sure why the implementation was so complicated in the first place; the test works fine with the simpler implementation.